NashvilleHistory/www/js/controllers/MarkersCtrl.js

```diff
@@ -7,6 +7,7 @@ app.controller('MarkersCtrl', function($scope, $state, $cordovaGeolocation, Auth
 
   // Holds data for displaying location markers on the Google map
   $scope.markers = [];
+  $scope.nearbyPlaces = [];
```

Comment:
I think this variable is unnecessary, we can use `$scope.$parent.MarkerCards` instead